WordPress/gutenberg
# Fix inspector tab active state border

## What?
The active border-bottom treatment to the `.block-editor-block-inspector__tab-item` in the new tabbed Styles/Settings UI is slightly off. That's because the elements surrounding the tabs typically have a border-bottom applied, whereas these tabs do not — instead each `.components-pane__body` element has a border-top. 

## Why?
To ensure consistency across all tabbed UX patterns. 

## How?
Minor style change. 

## Testing Instructions

 1. Turn on the relative Gutenberg experiment. 
 2. Open a Post or Page. 
 3. Insert a Paragraph Block.
 4. See the tabbed UI in the Inspector, with the active state correctly applied. 


## Screenshots or screencast 

Before: 

<img width="330" alt="CleanShot 2022-12-20 at 12 08 33@2x" src="https://user-images.githubusercontent.com/1813435/208725106-eca6ca25-7945-414a-8d8b-f235f44b74ef.png">

After: 

Notice the Page/Block and Style/Settings tabs look the same now.

<img width="323" alt="CleanShot 2022-12-20 at 12 08 53@2x" src="https://user-images.githubusercontent.com/1813435/208725167-ccea223a-93dd-4c23-bc08-d11df0b2a1c6.png">
